factom_keys/id.py

```diff
@@ -0,0 +1,197 @@
+import ed25519
+from bitcoin import base58
+from hashlib import sha256
+
+PREFIX_LENGTH = 3
+CHECKSUM_LENGTH = 4
+BODY_LENGTH = 35
+TOTAL_LENGTH = 39
+
+
+def generate_id_pairs():
+    """
+    Generate a set of Factom ID keypairs, sk1-sk4 and id1-id4.
+
+    :return: a list of tuples containing a random id key pair: (IDPrivateKey, IDPublicKey)
+    """
+    pairs = []
+
+    for i in range(0, 4):
+        signer, verifier = ed25519.create_keypair()
+        pair = (IDPrivateKey(signer.to_seed()), IDPublicKey(verifier.to_bytes()))
+        pairs.append(pair)
+
+    return pairs
```

Comment:
Rather than generate a list of pairs, I think this function should take the form of: 
```python
def generate_key_pair(level: int):
```
And instantiate the keys with the appropriate level prefix (if its valid). That way you only generate one key pair at the priority level that you want

Reply:
Yeah I agree this makes more sense for the general case. Does it make sense to still have a helper function that generates a complete set of keys? I think it's a common use case for a developer to want to create a complete set of identity keys (sk1, id1) - (sk4, id4).